apps/kitchen-sink/src/ensemble/screens/widgets.yaml

```diff
@@ -686,19 +689,30 @@ View:
               - Dropdown:
                   id: myDropdown
                   label: With Default Value
-                  value: 1
-                  styles:
-                    dropdownBackgroundColor: "aqua"
-                    dropdownBorderWidth: 1
-                    dropdownBorderColor: "lime"
-                    backgroundColor: "yellow"
-                    selectedBackgroundColor: "gray"
-                    selectedTextColor: "purple"
-                  items:
-                    - label: Option 1
-                      value: 1
-                    - label: Option 2
-                      value: 2
+                  panel:
+                    Form:
+                      children:
+                        - Date:
+                            label: Start Date
+                            id: testDate
+                            onChange:
+                              executeCode: |
+                                console.log(testDate)
+
+                        - Date:
+                            label: End Date
+                            id: testDate1
+                            onChange:
+                              executeCode: |
+                                console.log(testDate1)
+
+                        - Button:
+                            label: Apply
+                            onTap:
+                              executeCode: |
+                                const x = testDate.value;
+                                const y = testDate1.value;
+                                myDropdown.close()
```

Comment:
Can you also confirm you can set the value of the dropdown from here? And the dropdown shows the proper value?

Reply:
@evshi we can set value of dropdown something like, 
`myDropdown.setValue('value')`
or
`myDropdown.setSelectedValue('value')`

and we can get dropdown value like this way

`myDropdown.selectedValue`.

and yes, dropdown also shows the proper value